crates/fluvio-cli/src/client/core/unregister.rs

```diff
@@ -0,0 +1,24 @@
+use anyhow::Result;
+use clap::Parser;
+use fluvio_sc_schema::remote::RemoteSpec;
+use std::sync::Arc;
+use fluvio_extension_common::target::ClusterTarget;
+use fluvio_extension_common::Terminal;
+use super::get_admin;
+
+#[derive(Clone, Debug, Parser)]
+pub struct UnregisterOpt {
+    pub name: String,
+}
+impl UnregisterOpt {
+    pub async fn execute<T: Terminal>(
+        self,
+        _out: Arc<T>,
+        cluster_target: ClusterTarget,
+    ) -> Result<()> {
+        let admin = get_admin(cluster_target).await?;
+        admin.delete::<RemoteSpec>(&self.name).await?;
+        println!("edge cluster {:?} was unregistered", self.name);
```

Comment:
I changed for this, to keep the quotes, but use display instead of debug, what do you think?

```
println!("edge cluster \"{}\" was registered", self.name);
```

```
println!("edge cluster \"{}\" was unregistered", self.name);
```